### PR TITLE
exit() & _Exit() Functions in C++

### PR DESCRIPTION
**Pull request by ritika728**
<hr>

## Description 📜

I have added information about exit() and _Exit() function in C++ with proper examples and explanation in document form  # <2548> 

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] C/CPP

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

<hr>

### CODE REVIEW

**Comment by ritika728** [comment]
@Shivani0701 please review my PR.

File: C_CPP/Fundamentals/Functions & Recursion and its types/exit()&_Exit()Functions.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+* It takes an integer as an argument.
+***
+## SYNTAX:  
+exit(exit_code);  
+---where exit code is defining exit status.  
+For example:
+```
+exit(0)
+```
+***
```
Comment (vasu-1) [inline C_CPP/Fundamentals/Functions & Recursion and its types/exit()&_Exit()Functions.md:14]:
```suggestion
## SYNTAX:  
    exit(exit_code);  
- where exit code is defining exit status.  
For example:
```c++
exit(0)
```
***
```

File: C_CPP/Fundamentals/Functions & Recursion and its types/exit()&_Exit()Functions.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+* This function returns the control to the host environment.
+* **You must be wondering that what is the need for this when we have exit() function to terminate the process?**  🤔  
+--->So, the answer is: While using exit() function, there is some cleanup function like buffer removal...but _Exit() function will not perform any buffer removal or any type of cleanup and will terminate the program immediately.
+* Just like the exit function, remaining code will not be executed after this function is executed.
+```
+successful termination: exit code of 0/ EXIT_SUCCCESS
+unsuccessful termination: exit code of 1/ EXIT_FAILURE 
+```
+exit(0) indicates a clean exit without any error.  
+exit(1) indicates an error and that's why the program is terminating.
```
Comment (vasu-1) [inline C_CPP/Fundamentals/Functions & Recursion and its types/exit()&_Exit()Functions.md:83]:
```suggestion
- _Exit(0) indicates a clean exit without any error.  
- _Exit(1) indicates an error and that's why the program is terminating.  
```

File: C_CPP/Fundamentals/Functions & Recursion and its types/exit()&_Exit()Functions.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+    return 0;
+}
+```
+### OUTPUT:
+```
+(no output)
+```
+**Wondering why there is no output? Look at this example:**
+```
+#include<iostream>
```
Comment (vasu-1) [inline C_CPP/Fundamentals/Functions & Recursion and its types/exit()&_Exit()Functions.md:103]:
```suggestion
```c++
#include<iostream>
```

File: C_CPP/Fundamentals/Functions & Recursion and its types/exit()&_Exit()Functions.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+```
+successful termination: exit code of 0/ EXIT_SUCCCESS
+unsuccessful termination: exit code of 1/ EXIT_FAILURE 
+```
+exit(0) indicates a clean exit without any error.  
+exit(1) indicates an error and that's why the program is terminating.  
+--->Both report the status of termination of the program to the host environment.  
+Any other integer (except 0) will also indicate unsuccessful termination.
+```
+#include<iostream>
```
Comment (vasu-1) [inline C_CPP/Fundamentals/Functions & Recursion and its types/exit()&_Exit()Functions.md:87]:
```suggestion
```c++
#include<iostream>
```

File: C_CPP/Fundamentals/Functions & Recursion and its types/exit()&_Exit()Functions.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+successful termination: exit code of 0/ EXIT_SUCCESS
+unsuccessful termination: exit code of 1/ EXIT_FAILURE 
+```
+exit(0) indicates a clean exit without any error.  
+exit(1) indicates an error and that's why the program is terminating.  
+--->Both report the status of termination of the program to the host environment.  
+Any other integer (except 0) will also indicate unsuccessful termination. 
+***
+```
+#include <iostream>
```
Comment (vasu-1) [inline C_CPP/Fundamentals/Functions & Recursion and its types/exit()&_Exit()Functions.md:28]:
```suggestion
```c++
#include <iostream>
```

File: C_CPP/Fundamentals/Functions & Recursion and its types/exit()&_Exit()Functions.md [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+***
+* When the program terminates,this value gets returned to the operating system and not to the caller.
+* Remaining code will not be executed after this function is executed.
+* Objects that have static storage duration, are dismantled and files created by tmpfile function are removed when exit function executes.
+```
+successful termination: exit code of 0/ EXIT_SUCCESS
+unsuccessful termination: exit code of 1/ EXIT_FAILURE 
+```
+exit(0) indicates a clean exit without any error.  
+exit(1) indicates an error and that's why the program is terminating.
```
Comment (vasu-1) [inline C_CPP/Fundamentals/Functions & Recursion and its types/exit()&_Exit()Functions.md:23]:
```suggestion
- exit(0) indicates a clean exit without any error.  
- exit(1) indicates an error and that's why the program is terminating.  
```

**Review by vasu-1 — CHANGES_REQUESTED** [review]
Good Content !!! @ritika728 
Just Require Some Markdown Formatting !!!
Do this changes And also some others are required then do them also !

**Comment by ritika728** [comment]
@vasu-1 I have made the required changes. Please review!

**Review by apeksha235 — APPROVED** [review]
@Suvraneel please merge this

**Pull request merged**